### PR TITLE
Mark the query frequency and average duration QueryStats metrics as deprecated.

### PR DESCRIPTION
**Pull request by andreidan**
## Checklist

 - [ ] User relevant changes are recorded in ``CHANGES.txt``
 - [ ] Touched code is covered by tests
 - [ ] Documentation has been updated if necessary
 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by andreidan** [comment]
@mfussenegger created a ``Deprecrations`` section

File: blackbox/docs/appendices/release-notes/unreleased.rst [hunk 1]
```diff
@@ -43,6 +43,13 @@ Breaking Changes
 
 None
 
+Deprecations
```
Comment (mfussenegger) [inline blackbox/docs/appendices/release-notes/unreleased.rst]: In https://github.com/crate/crate/blob/master/blackbox/docs/appendices/release-notes/3.2.0.rst#id20 we had deprecations as a sub-category of changes. Not sure which structure is better.


This https://github.com/crate/crate/pull/8246/files#diff-8918b36d12905ff85de79bcbdcf052aaR56 here and https://github.com/crate/crate/pull/8246/files#diff-8918b36d12905ff85de79bcbdcf052aaR78 is also a deprecation that we could put into this section. (maybe there are more that I missed, please double check)

Comment (andreidan) [reply blackbox/docs/appendices/release-notes/unreleased.rst]: pushed another update. I  think having a standalone section for deprecations is more visible but I'm also happy to move it below changes.

Comment (andreidan) [reply blackbox/docs/appendices/release-notes/unreleased.rst]: @joemoe any opinions about the deprecations being a standalone category or a sub-category of Changes?

Comment (joemoe) [reply blackbox/docs/appendices/release-notes/unreleased.rst]: @andreidan we should depend that on the importance and amount of deprecations. in this we will have a lot so i'd make it a dedicated category.

**Pull request merged**